source/time_tables/departure_boards.cpp

```diff
@@ -85,13 +85,56 @@ render(PbCreator& pb_creator,
                 }
             }
         }
-        const auto& it = response_status.find(id_vec.first.second.val);
+        const auto& it = response_status.find(id_vec.first);
         if(it != response_status.end()){
             schedule->set_response_status(it->second);
         }
     }    
 }
 
+static time_duration to_navitia (const boost::posix_time::time_duration& dur) {
+    return navitia::seconds(dur.total_seconds());
+}
+
+time_duration length_of_time(const time_duration& duration_1,
+                       const time_duration& duration_2) {
+    if (duration_1 <= duration_2) {
+        return duration_2 - duration_1;
+    } else {
+        return duration_2 + (hours(24) - duration_1);
+    }
+}
+
+bool between_opening_and_closing(const time_duration& me,
+                                 const time_duration& opening,
+                                 const time_duration& closing) {
+    if (opening < closing) {
+        return (opening <= me && me <= closing);
+    } else {
+        return (opening <= me || me <= closing);
+    }
+}
+
+bool line_closed (const time_duration& duration,
+                  const time_duration& opening,
+                  const time_duration& closing,
+                  const pt::ptime& date ) {
```

Comment:
very minor comment, but you shouldn't put a space char between the variable name and the `)`